package.json

```diff
@@ -27,7 +27,8 @@
     "docker:up": "yarn docker:compose up",
     "docker:stop": "yarn docker:compose stop",
     "docker:down": "yarn docker:compose down",
-    "docker:build": "yarn docker:compose build",
+    "docker:build": "docker run -it --rm  -v ${PWD}:/usr/src/app -w /usr/src/app node yarn",
```

Comment:
Do we need to run `yarn build` here? For now, it just downloads the dependencies.

Reply:
You're right. Addressing now